packages/react/src/popover-content/PopoverContent.css.ts

```diff
@@ -0,0 +1,58 @@
+import { keyframes, recipe, style } from "../vanilla-extract";
+
+const slideUpAndFade = keyframes({
+  "0%": { opacity: 0, transform: "translateY(2px)" },
+  "100%": { opacity: 1, transform: "translateY(0)" },
+});
+
+const slideRightAndFade = keyframes({
+  "0%": { opacity: 0, transform: "translateX(-2px)" },
+  "100%": { opacity: 1, transform: "translateX(0)" },
+});
+
+const slideDownAndFade = keyframes({
+  "0%": { opacity: 0, transform: "translateY(-2px)" },
+  "100%": { opacity: 1, transform: "translateY(0)" },
+});
+
+const slideLeftAndFade = keyframes({
+  "0%": { opacity: 0, transform: "translateX(2px)" },
+  "100%": { opacity: 1, transform: "translateX(0)" },
+});
+
+export const content = recipe({
+  base: [
+    {
+      border: "1",
+      borderColor: "border.secondary",
+      display: "flex",
+      flexDirection: "column",
+      gap: "2",
+      p: "sm",
+      rounded: "lg",
+      shadow: "md",
```

Comment:
let's add z-index here:
```suggestion
      shadow: "md",
      z: "popover",
```